pkg/cmd/run/view/view.go

```diff
@@ -212,13 +212,31 @@ func runView(opts *ViewOptions) error {
 		return fmt.Errorf("failed to get run: %w", err)
 	}
 
+	if opts.Exporter != nil {
+		if len(run.Jobs) == 0 && contains(opts.Exporter.Fields(), "jobs") {
```

Comment:
I added a check here so we can save an api call in some cases. A field `jobs` could be a constant in `shared` package but it looks weird when other fields are not.